### PR TITLE
Use correct dashboard definition URL

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: voxxed-microservices-2018/module-01.adoc [hunk 1]
```diff
@@ -295,7 +295,7 @@ After "Add" is clicked, Grafana will test the connection to the data source.
 image::grafana_prometheus_data_source.png[grafana prometheus data source]
 
 From the top left menu, click on "Dashboards" and then "Import" to open the "Import Dashboard" window.
-Open a browser tab and navigate to `https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/0.7.0/metrics/examples/grafana/kafka-dashboard.json`.
+Open a browser tab and navigate to `https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/grafana/strimzi-kafka.json`.
```
Comment (gunnarmorling) [inline voxxed-microservices-2018/module-01.adoc]: Hum, hum, I'm still seeing no data in Grafana :( Also the drop-downs for selecting namespace etc. are empty :(

Comment (jpechane) [reply voxxed-microservices-2018/module-01.adoc]: Thanks a lot, I forgot to add one more step!

File: voxxed-microservices-2018/module-01.adoc [hunk 2]
```diff
@@ -253,11 +253,20 @@ $ export NAMESPACE=voxxed
...
-Finally, create the Prometheus service by running:
+Create the Prometheus service by running:
 
 [source,sh]
 $ oc apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/prometheus/kubernetes.yaml
 
+Finally it is necessary to enable metrics collection from Kafka brokers by definining a set of data available from JMX.
+This is configured in the Kafka resource.
+
+ * Open the kafka resource using command `oc edit kafka my-cluster`.
```
Comment (gunnarmorling) [inline voxxed-microservices-2018/module-01.adoc]: Pasting this in seems a bit brittle and easy to get wrong. How about providing the entire resource and simply overwrite the existing one?

Comment (jpechane) [reply voxxed-microservices-2018/module-01.adoc]: I was thinking about patching but I've not found yet a way how to patch from a file

Comment (jpechane) [reply voxxed-microservices-2018/module-01.adoc]: @gunnarmorling How about something like `oc get kafka my-cluster -o yaml | sed '/kafka:/ r kafka-metrics.yaml' - | oc apply -f -`

**Comment by gunnarmorling** [comment]
Applied. Thanks a lot, happy to see this working finally :) Added one more Trello card though as some of the metrics are still empty (not too important for our purposes, though).

**Pull request closed without merging**